### ISSUE TITLE
List of challenge set papers (and other behavioral studies) to add

### ISSUE THREAD

**Issue by boknilev** [boknilev/nlp-analysis-methods#1]
This is a list of potential papers to add that describe challenge sets (Section 4 of the paper, Table SM2 in the supplementary materials/website). 
- [Grammatical Analysis of Pretrained Sentence Encoders with Acceptability Judgments](https://arxiv.org/pdf/1901.03438.pdf)
- PROTEST
  - [PROTEST: A Test Suite for Evaluating Pronouns in Machine Translation](https://pdfs.semanticscholar.org/eb84/b6ec16d37d58547b871bf5d5e2f40d07ba2f.pdf)
  - [A Pronoun Test Suite Evaluation of the English--German MT Systems at WMT 2018](http://aclweb.org/anthology/W18-6435)
  - [Pronoun Translation in English–French Machine Translation: An Analysis of Error Types](https://arxiv.org/pdf/1808.10196.pdf)
  - [A Graphical Pronoun Analysis Tool for the PROTEST Pronoun Evaluation Test Suite](http://www.aclweb.org/anthology/W16-3418)
- Item Response Theory
  - [Building an Evaluation Scale using Item Response Theory](https://arxiv.org/pdf/1605.08889.pdf)
  - [Understanding Deep Learning Performance through an Examination of Test Set Difficulty: A Psychometric Case Study](https://arxiv.org/pdf/1702.04811.pdf)
- [Still a Pain in the Neck: Evaluating Text Representations on Lexical Composition](https://arxiv.org/abs/1902.10618)
- “Inoculation by Fine-Tuning: A Method for Analyzing Challenge Datasets”: A method for analyzing challenge sets, NAACL 2019
- https://github.com/boknilev/nlp-analysis-methods/issues/6
- [Neural Language Models as Psycholinguistic Subjects: Representations of Syntactic State](https://arxiv.org/abs/1903.03260)
- [Structural Supervision Improves Learning of Non-Local Grammatical Dependencies](https://arxiv.org/abs/1903.00943)
- [Right for the Wrong Reasons: Diagnosing Syntactic Heuristics in Natural Language Inference](https://arxiv.org/abs/1902.01007)
- [Evaluating Gender Bias in Machine Translation](https://arxiv.org/pdf/1906.00591.pdf)
- [What BERT is not: Lessons from a new suite of psycholinguistic diagnostics for language models](https://arxiv.org/abs/1907.13528)
- [A Test Suite and Manual Evaluation of Document-Level NMT at WMT19](https://arxiv.org/abs/1908.03043)
- EMNLP 2019: Evaluating Pronominal Anaphora in Machine Translation: An Evaluation Measure and a Test Suite
- [Does BERT agree? Evaluating knowledge of structure dependence through
 agreement relations](https://arxiv.org/abs/1908.09892)
- [The compositionality of neural networks: integrating symbolism and
 connectionism](https://arxiv.org/abs/1908.08351)
- [Probing Natural Language Inference Models through Semantic Fragments](https://arxiv.org/abs/1909.07521)
- CoNLL 2019: [Using Priming to Uncover the Organization of Syntactic Representations
 in Neural Language Models](https://arxiv.org/abs/1909.10579)
- [What BERT is not: Lessons from a new suite of psycholinguistic diagnostics for language models](https://arxiv.org/abs/1907.13528)
- [Probing What Different NLP Tasks Teach Machines about Function Word Comprehension](https://arxiv.org/abs/1904.11544)
- [Linguistic evaluation of German-English Machine Translation using a Test Suite](https://arxiv.org/abs/1910.07457)
- [Adversarial Analysis of Natural Language Inference Systems](https://arxiv.org/abs/1912.03441)
- [Diversify Your Datasets: Analyzing Generalization via Controlled Variance in Adversarial Datasets](https://arxiv.org/abs/1910.09302)
- [Testing Machine Translation via Referential Transparency](https://arxiv.org/pdf/2004.10361.pdf)
- [The Curse of Performance Instability in Analysis Datasets: Consequences, Source, and Suggestions](https://arxiv.org/abs/2004.13606)
- [NoiseQA: Challenge Set Evaluation for User-Centric Question Answering](https://arxiv.org/pdf/2102.08345.pdf)
- [Analysis and Evaluation of Language Models for Word Sense Disambiguation](https://watermark.silverchair.com/coli_a_00405.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAqswggKnBgkqhkiG9w0BBwagggKYMIIClAIBADCCAo0GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM9tmJxyMWINGWiUWTAgEQgIICXjMlvsVZg5RpjtbFyFDQHxeu4qQh45K53Ebv-71JuyYNk336ADL1WgTMmEXkD8dmVywV0Vu2lYt5Tq9cn9GYP9BKet8VG-CiVnSu4c0WhAbIYiX9LT2_-dQmzM68joJBK6fLLRloTVxksDXODQTA_RScz_XotrNzSIzu4YmGcoPFCZpiXhkqnq8_35TPCWqkbbWBMDW4j7WUkzvxRcbkY1iKdOxw5uHBXrK9ry6aKcDvtcJWu2wOT30w6X8-T9R-TD7Wki4y7t0kH-A9JF7HgKd8Kz0lrAHC2wfkF1lFJS1wM8KxP0IRHJbaU8VWUrWHzNzdJafnWJDwyedZqrx8ONOk9JkcbH5Ci2nV4DB-QUNv9KXUCbxVRUMCAfXerbYPe_qADMFopxRCnAbAvbIO41f8rg3F86p1MeGcmXuv4cEQ1vqu87cQ-3zk-QLOXKNeKQ8Dirxgw_7N7j2AQ2RRlxDWOWRndzlblBcodtXGg-7JS3RJgBiHLmfXqUlULpPt27Hlu1q4wru9PguAMrLlRvcHJuM39Fs2flnpCx7cuFEHmqgBpjyhmDG8rw_jkslUTKDbEQA-vgw6H14ptYIPMwaUIn0u9UK-1wn_NpJENRPh9XgHN6LQbXPTbl_w7joghRXM5QBgmWRK6TErmGSu1Jj0pO2nfkBzX4Oh_AnIJTKoD5LacPtdwMtnKkHwZM4C6A2OD0fIX3M6cVZF4u9TbBSf9kj_apSUV4tEn-Y88eQXXGeLjfju2KxBe3AzG-TfrBpeAXc0bnKHrwjz99-upcG3HkQFuT-iZKojyFjrog)
